docs/aarch64_build.md

```diff
@@ -0,0 +1,90 @@
+# Building IA2 and compartmentalized programs for AArch64
+
+Because the AArch64 port of IA2 requires changes to compiler code generation
+(reserving the x18 register and instrumenting memory accesses),
+all code executing in a compartmentalized process must be built with a modified toolchain.
+
+To do so, the first step is to build the modified toolchain itself.
+The toolchain is available at <https://github.com/immunant/llvm-project/tree/fw/instrument-llvm-ir>,
+and can be built by running the `build.sh` script provided.
```

Comment:
```suggestion
and can be built by running the [`build.sh`](https://github.com/immunant/llvm-project/blob/fw/instrument-llvm-ir/build.sh) script provided.
```

Reply:
Yes, unfortunately it is. That's because it gets used to compile some non-AArch64 targets when building runtime libs due to how LLVM's build system works.